pioneer/packages/joy-roles/src/transport.substrate.ts

```diff
@@ -256,27 +249,19 @@ export class Transport extends TransportBase implements ITransport {
   }
 
   protected async areGroupRolesOpen (group: WorkingGroups, lead = false): Promise<boolean> {
-    const nextId = await this.cachedApiMethodByGroup(group, 'nextOpeningId')() as GroupOpeningId;
-
-    // This is chain specfic, but if next id is still 0, it means no openings have been added yet
-    if (nextId.eq(0)) {
-      return false;
-    }
-
-    const groupOpenings = new MultipleLinkedMapEntry<GroupOpeningId, GroupOpening>(
-      OpeningId,
-      workingGroupsApiMapping[group].openingType,
-      await this.cachedApiMethodByGroup(group, 'openingById')()
+    const groupOpenings = await this.entriesByIds<GroupOpeningId, GroupOpening>(
+      this.apiMethodByGroup(group, 'openingById')
     );
 
-    for (const groupOpening of groupOpenings.linked_values) {
+    for (const [/* id */, groupOpening] of groupOpenings) {
```

Comment:
If we don't need the key (id), which we get from value.hiring_opening_id.id, perhaps better to get `.values()` from map only instead of `.entries()`. Its a minor point, but it would give a cleaner Destructuring assignment

Reply:
I dind't notice `.values()` method to be available. We could write an implementation similar to `entiresByIds`, but it would actually do the same thing (execute `entiresByIds` and then throw away ids), since we still need ids to sort the entries (by default they are returned in unpredictable order)